src/styles/main.scss

```diff
@@ -1 +1,113 @@
-// put code here
+
+html {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+}
+
+.clock {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  position: relative;
+  width: 500px;
+  height: 500px;
+
+  border-radius: 50%;
+
+  &__second-hand {
+    border-radius: 5px;
+    position: absolute;
+
+    transform: translateY(-50%);
+
+    height: 50%;
+    width: 8px;
+    background-color: rgb(253, 200, 20);
+    z-index: 6;
+
+    animation: rotateHand 60s linear infinite;
+  }
+
+  &__minute-hand {
+    height: 25%;
+    width: 15px;
+    position: absolute;
+    background-color: rgb(191, 209, 130);
+    z-index: 5;
+    
+    animation: rotateHand 3600s linear infinite;
```

Comment:
Need to fix the time for rotate hands as on the task description
<img width="496" alt="image" src="https://github.com/mate-academy/layout_stop-watch/assets/64750958/70462a6f-dfac-4a1b-a930-b4ae9ebcbe5d">

Reply:
If I understood correctly instruction, instead of 60 and 3600s for seconds-hand and minute-hand, should I use 5s and 60s?